### PR TITLE
Update docs

### LINKED ISSUE #79: TERRA-53 ⁃ Documentation: astra_database, needs more details on required attributes
### Terraform Version
Terraform v1.1.5
on linux_amd64

### Affected Resource(s)
- astra_database

The official TF doc: https://registry.terraform.io/providers/datastax/astra/latest/docs/resources/database needs more guidance on the description and usage of the required attributes. Example:

- `cloud_provider`: what are the list of valid values? 
- `keyspace`: Why does this DB needs that exact keyspace?
- `regions`: list of valid values? Would be helpful to give an example of CLI query to get the list of valid values. For example, if the Cloud provider is Azure. Maybe the list of valid regions could be `az account list-locations --query "[?@.metadata.geographyGroup=='Canada'].{ displayName: displayName, name: name}"`

┆Issue is synchronized with this [Jira Task](https://datastax-oss.atlassian.net/browse/TERRA-53) by [Unito](https://www.unito.io)
┆friendlyId: TERRA-53
┆priority: Major

### LINKED ISSUE #80: TERRA-54 ⁃ Documentation: astra_role, what is the format of a DRN resource string and policy?
### Terraform Version
Terraform v1.1.5
on linux_amd64

### Affected Resource(s)
- astra_role

The official Terraform doc: https://registry.terraform.io/providers/datastax/astra/latest/docs/resources/role gives a succint example of two attributes

```
resources   = ["drn:astra:org:f9f4b1e0-4c05-451e-9bba-d631295a7f73"]
  policy    = ["db-all-keyspace-create"]
```

The documentation needs more details on:

- Describe the format of a DRN resource string. Give a couple of example of the possible variations of resource string
- What are the valid values for `policy` ?

┆Issue is synchronized with this [Jira Task](https://datastax-oss.atlassian.net/browse/TERRA-54) by [Unito](https://www.unito.io)
┆friendlyId: TERRA-54
┆priority: Major

### PR DESCRIPTION
**Pull request by emerkle826**
Fixes #79 
Fixes #80

### CODE REVIEW

File: internal/provider/resource_database.go [hunk 1]
```diff
@@ -51,22 +51,22 @@ func resourceDatabase() *schema.Resource {
 				ValidateFunc: validation.StringMatch(regexp.MustCompile("^.{2,}"), "name must be atleast 2 characters"),
 			},
 			"keyspace": {
-				Description:      "keyspace",
+				Description:      "Initial keyspace name. For additional kepyspaces, use the astra_keyspace resource.",
```
Comment (jeffbanks) [inline internal/provider/resource_database.go:54]: misspelling here

File: internal/provider/resource_role.go [hunk 2]
```diff
@@ -54,7 +54,7 @@ func resourceRole() *schema.Resource {
 			},
 
 			"policy": {
-				Description:  "List of policies for the role.",
+				Description:  "List of policies for the role. See https://docs.datastax.com/en/astra/docs/user-permissions.html#_operational_roles_detail for spported policies.",
```
Comment (jeffbanks) [inline internal/provider/resource_role.go:57]: misspelling here

**Review by jeffbanks — APPROVED** [review]
looks good overall -- just a couple of misspellings, then good to merge.

**Review by jeffbanks — APPROVED** [review]
All good!

**Comment by emerkle826** [comment]
Thanks @jeffbanks

**Pull request merged**